package.json

```diff
@@ -26,16 +26,17 @@
     "release": "standard-version"
   },
   "dependencies": {
-    "@aeternity/aepp-sdk": "^11.0.1",
+    "@aeternity/aepp-sdk": "^13.0.0-alpha.1",
```

Comment:
I would finish this PR without FF support based on 13.0.0-alpha.0 and then update to 13.0.0-alpha.1, too many changes otherwise

Reply:
Sure